### DISCUSSION TITLE
Bot ratings reset to zero?

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#20] [Q&A] [answered]
I had [post](https://github.com/azadbolour/eagerwords-community/discussions/8#discussioncomment-5714607) on reset bot ratings in a different discussion but opened up a new post for tracking

**Comment by azadbolour** [comment, marked as answer]
I am guessing that you are referring to the fact that bot ratings are
coming out as zero in the browser app. If not, let me know the specifics.

The zeroes have now been fixed in the test version of the browser app and
will be deployed to the production version once testing is complete
(sometime in May).

Azad

**Reply by danconde** [reply]
That is correct - I used the test browser version and saw the zero ratings.

**Reply by danconde** [reply]
I used the test browser edition again, and could not replicate it, and saw non-zero ratings.

**Comment by azadbolour** [comment]
Thanks so much for verifying the correction.
